src/apiServices/AuthServiceNode.ts

```diff
@@ -0,0 +1,96 @@
+import { postRequest, getRequest } from '../http'
+
+import { loginUser } from '../auth/loginUser'
+
+import { getHeaders } from '../util/getHeaders'
+import { errorHandler } from '../util/errorHandler'
+import { API_URL_PRODUCTION } from '../util/constants'
+
+import { JwtTokensInterface } from '..'
+
+export class AuthServiceNode {
+
+    public ethAddress: string
+    public password: string
+    public jwtTokens: JwtTokensInterface = {
+        access: '',
+        refresh: ''
+    }
+
+    constructor(ethAddress?: string, password?: string) {
+        this.ethAddress = ethAddress as string
+        this.password = password as string
```

Comment:
Perhaps we can call `login()` in the constructor? What do you think? Would that make for a better experience?